code/client/src/pages/Create.jsx

```diff
@@ -45,108 +44,84 @@ const getGoogleAuthenticatorAppLink = (os) => {
 }
 
 const sectionViews = {
-  setupWalletDetails: 1,
+  setupWalletDetails: 1, // not used
   setupOtp: 2,
-  setupSecondOtp: 3,
-  prepareWallet: 4,
-  walletSetupDone: 5
+  prepareWallet: 3,
+  walletSetupDone: 4
 }
 
+const securityParameters = ONEUtil.securityParameters({
+  majorVersion: ONEConstants.MajorVersion,
+  minorVersion: ONEConstants.MinorVersion,
+})
+
 const Create = ({ expertMode, showRecovery }) => {
-  // eslint-disable-next-line no-unused-vars
-  const dev = useSelector(state => state.global.dev)
-  const { isMobile, os } = useWindowDimensions()
-  const dispatch = useDispatch()
-  const history = useHistory()
+  const { isMobile } = useWindowDimensions()
   const network = useSelector(state => state.global.network)
   const wallets = useSelector(state => state.wallet)
+  const generateNewOtpName = () => ONENames.genName(Object.keys(wallets).map(k => wallets[k].name))
+
+  // Store info collected across the creation process.
+  const walletInfo = useRef({
+    name: generateNewOtpName(),
+    seed: generateOtpSeed(),
+    seed2: generateOtpSeed(),
+    duration: WalletConstants.defaultDuration,
+    doubleOtp: false
+  }).current
 
   const [effectiveTime, setEffectiveTime] = useState()
+  const [otpReady, setOtpReady] = useState(false)
 
-  const generateNewOtpName = () => ONENames.genName(Object.keys(wallets).map(k => wallets[k].name))
-  const [name, setName] = useState(generateNewOtpName())
```

Comment:
Let's keep name. We should allow users to pick their own name and dynamically change the QR code

Reply:
ya but it's not used yet, and we can add it back whenever we need, like in the v2ui. But for now, I want to remove those unused stuff so we can keep it simple for the upgrade